packages/mjml-section/README.md

```diff
@@ -44,12 +44,12 @@ border-radius       | px          | border radius                  | n/a
 border-right        | string      | css border format              | n/a
 border-top          | string      | css border format              | n/a
 css-class           | string      | class name, added to the root HTML element created | n/a
-direction           | ltr / rtl   | set the display order of direct children | ltr
```

Comment:
this should stay unchanged

Reply:
Action: Substituted the `9d` version of `direction` for the `next` version.

* An aside: Both forms are technically correct ("ltr" and "rtl" are both strings). I consider the above form preferable because only certain strings are meaningful. There are other places in the documentation with this condition. I'll do a PR for those at some point and see how the other ideas appeal.